### PR TITLE
[backport] Fix `filename` arg of `PlotReport`

### PR DESCRIPTION
**Pull request by toslunar**
Backport of #6866

### CODE REVIEW

**Comment by toslunar** [comment]
[automatic post] Jenkins, test this please.

File: chainer/training/extensions/variable_statistics_plot.py [hunk 1]
```diff
@@ -187,17 +187,15 @@ def __init__(self, targets, max_sample_size=1000,
                  plot_mean=True, plot_std=True,
                  percentile_sigmas=(
                      0, 0.13, 2.28, 15.87, 50, 84.13, 97.72, 99.87, 100),
-                 trigger=(1, 'epoch'), filename='statistics.png',
+                 trigger=(1, 'epoch'), filename=None,
```
Comment (niboshi) [inline chainer/training/extensions/variable_statistics_plot.py]: I think this causes an unwanted change in the documentation.

Comment (niboshi) [reply chainer/training/extensions/variable_statistics_plot.py]: I'll address it in another PR.

**Comment by niboshi** [comment]
LGTM

File: chainer/training/extensions/variable_statistics_plot.py (hunk 1 above)
Comment (niboshi) [reply chainer/training/extensions/variable_statistics_plot.py]: #6932

**Pull request merged**